website/docs/docs/deploy/job-notifications.md

```diff
@@ -1,33 +1,79 @@
 ---
 title: "Job notifications"
 id: "job-notifications"
-description: "Set up notifications in dbt Cloud to receive Email or Slack alerts for job run status."
+description: "Set up notifications in dbt Cloud to receive email or Slack alerts about job run status."
 ---
 
+Set up notifications in dbt Cloud to receive email or Slack alerts when a job run succeeds, fails, or is cancelled.
 
-Setting up notifications in dbt Cloud will allow you to receive alerts via Email or a chosen Slack channel when a job run succeeds, fails, or is cancelled.
+## Email notifications
 
-### Email
+You can receive email alerts by configuring the dbt Cloud notification settings. As a **user**, you can set up email notifications for yourself. As an **admin**, you can set up notifications for yourself and other team members. Refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users) for details on license types that are eligible for email notifications.
 
-These are the following options for setting up email notifications. Refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users) for info on license types eligible for email notifications. 
```

Comment:
i added a prerequisites section for the `Slack notifications` section below. 

are there prereqs for email notifications that also need to be added here?